index.html

```diff
@@ -67,7 +67,7 @@ <h2>Registration Entry Requirements and Update Process</h2>
 			<li>Each entry should include a link that references a public available
 			specification (PAS) that defines the associated interface type.</li>
 			<li>Each entry must include contact information of the requestor.</li>
-                        <li>Each entry must include an <dfn><code>availableFromTimeline</code></dfn> boolean
+                        <li>Each entry must include an <dfn export><code>availableFromTimeline</code></dfn> boolean
```

Comment:
So, to export we need two things: 

 1. a consumer: which spec needs this? 
 2. a `data-dfn-type`... see the types at https://respec.org/xref/ 

If this is an IDL definition, it should be automagically exported. Otherwise, can you choose an type appropriate?

Reply:
https://w3c.github.io/performance-timeline/#filter-buffer-map-by-name-and-type consumes this, although it's currently bugged. It's not an IDL thing, so maybe `_CONCEPT_`? Is this type the only non-IDL one?

Reply:
Sorry, trying to find the types list... I thought they were a drop down on respec.org/xref... but not. One sec...

Reply:
https://tabatkins.github.io/bikeshed/#dfn-types

Reply:
It might just be "dfn"... if so, check if ReSpec is already adding that automatically by default.

Reply:
>check if ReSpec is already adding that automatically by default.

Yes, ReSpec automatically does that.

Reply:
Such good software. Must have good maintainers! 🤣